Submodels/Parameterizations.py

```
@@ -414,26 +499,246 @@ def ConvertBezier2AirfoilCoordinates(self,
+        
+        else:
+            # If camber is zero, handle appropriately
+            bezier_camber = np.zeros_like(bezier_thickness)
+            bezier_camber_x = bezier_thickness_x
+
+        return bezier_thickness, bezier_thickness_x, bezier_camber, bezier_camber_x
+
     def FindInitialParameterization(self, 
                                     reference_file: str,
-                                    plot: bool = False) -> tuple[float, float, float, float, float, float, float, float, float]:
+                                    plot: bool = False) -> tuple[np.ndarray, dict]:
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)